src/test/java/com/attica/athens/domain/agora/AgoraOpenApiIntegrationTest.java

```diff
@@ -0,0 +1,159 @@
+
+            // when & then
+            mockMvc.perform(get("/{prefix}/agoras", API_V1_OPEN)
+                            .param("status", request.status())
+                            .param("category", request.category() != null ? request.category().toString() : "")
+                            .param("next", request.next() != null ? request.next().toString() : ""))
+                    .andExpect(status().isBadRequest())
+                    .andExpectAll(
+                            jsonPath("$.success").value(false),
+                            jsonPath("$.response").value(nullValue()),
+                            jsonPath("$.error.code").value(expectedCode),
+                            jsonPath("$.error.message").value(expectedMessage)
+                    );
+        }
+
+        @Test
+        @DisplayName("카테고리로 아고라를 조회한다.")
+        void getAgoraByCategorySortByIdDesc() throws Exception {
+            // given
+            AgoraRequest request = new AgoraRequest("active", 1L, null);
+
+            // when & then
+            mockMvc.perform(get("/{prefix}/agoras", API_V1_OPEN)
+                            .param("status", request.status())
+                            .param("category", request.category().toString())
+                            .param("next", request.next() != null ? request.next().toString() : ""))
+                    .andExpect(status().isOk())
+                    .andExpectAll(
+                            jsonPath("$.success").value(true),
+                            jsonPath("$.response.agoras.length()").value(2),
+                            jsonPath("$.response.agoras[0].id").value(2),
+                            jsonPath("$.response.agoras[0].agoraTitle").value("Tools"),
+                            jsonPath("$.response.agoras[0].agoraColor").value("Red"),
+                            jsonPath("$.response.agoras[0].participants.pros").value(0),
+                            jsonPath("$.response.agoras[0].participants.cons").value(0),
+                            jsonPath("$.response.agoras[0].participants.observer").value(0),
+                            jsonPath("$.response.agoras[0].createdAt").value("2023-09-13T14:12:15"),
+                            jsonPath("$.response.agoras[0].status").value("QUEUED"),
+                            jsonPath("$.response.agoras[1].id").value(1),
+                            jsonPath("$.response.agoras[1].agoraTitle").value("Games"),
+                            jsonPath("$.response.agoras[1].agoraColor").value("Green"),
+                            jsonPath("$.response.agoras[1].participants.pros").value(0),
+                            jsonPath("$.response.agoras[1].participants.cons").value(0),
+                            jsonPath("$.response.agoras[1].participants.observer").value(0),
+                            jsonPath("$.response.agoras[1].createdAt").value("2024-04-01T23:14:09"),
```

Comment:
p2: 이 코드는 테스트 데이터가 변경되면 영향을 받을 것 같아요! 이 코드는 어떨까요?
```suggestion
                            jsonPath("$.response.agoras[0].createdAt").value(matchesPattern("\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}"))
```